star-navigation/src/utils/sim-utils.ts

```diff
@@ -73,6 +73,13 @@ export const getStarPositionAtTime = (options: { starHip: number, celestialSpher
   return starPos;
 };
 
+export const toPositiveHeading = (heading: number) => {
+  if (heading < 0) {
+    return -heading;
+  }
+  return 360 - heading;
```

Comment:
Should this be modulus 360 to match the code the simulation view?

Reply:
True, thanks. I've fixed that and added some comments why we need so many `% 360`. 😉